src/views/download/download.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../colors";
+@import "../../frameless";
+
+$developer-spot: $ui-yellow;
+
+#view {
+    padding: 0;
+}
+
+.download {
+    .title-banner {
+        &.masthead {
+            background-color: $developer-spot;
+            padding-bottom: 0;
+
+            h1 {
+                margin: 0 0 2rem 0;
+                text-align: left;
+                color: $ui-white;
+            }
+
+            p {
+                margin: 0;
+                width: $cols6;
+                text-align: left;
+                color: $ui-white;
+
+                a {
+                    border-bottom: 1px solid $ui-white;
+                    color: $ui-white;
+                }
+            }
+
+            .band {
+                $band-color: hsla(55, 100, 70, .15);
+                
+                margin-top: 2rem;
+                background-color: $band-color;
+                padding: 1rem 0;
+            }
+
+            .sub-nav {
+                text-align: left;
+                justify-content: flex-start;
+
+                li {
+                    margin: 0 .5rem 0 0;
+                }
+            }
+        }
+    }
+    
+    .download-content {
+        padding-bottom: 2rem;
+    }
+    
+    .installation {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: flex-start;
+        justify-content: space-between;
+    }
+        
+    .installation-column {
+        max-width: $cols4;
+    }
+    
+    .installation-downloads {
+        list-style: none;
```

Comment:
this also needs to remove the default padding for a `ul` element – `padding-left: 0` should do it, see the indents on the list items:
<img width="930" alt="screen shot 2017-05-23 at 2 04 08 pm" src="https://cloud.githubusercontent.com/assets/1815772/26368875/f5539bf0-3fc0-11e7-9523-71c975a34d11.png">